easybuild/tools/systemtools.py

```diff
@@ -106,6 +107,28 @@ def get_core_count():
     _log.nosupport("get_core_count() is replaced by get_avail_core_count()", '2.0')
 
 
+def get_total_memory():
+    """
+    Try to ascertain this node's total memory
+    
+    @return: total memory as an integer, specifically a number of megabytes
+    """
+    memtotal = None
+    os_type = get_os_type()
+
+    if os_type == LINUX and os.path.exists(PROC_MEMINFO_FP):
+        with open(PROC_MEMINFO_FP, 'r') as meminfo:
+            for memline in meminfo:
+                mem_mo = re.match(r'MemTotal:\s*(\d+)\s*kB$', memline)
+                if mem_mo:
+                    memtotal = int(mem_mo.group(1)) / 1024
+
+    if memtotal is None:
```

Comment:
Allowing $BC_MEM_PER_NODE would permit to overcome the limitation on missing or unmounted /proc/meminfo,
so that people that hit this conditional may pass it succesfully...